frontend/src/components/app/app.tsx

```diff
@@ -1,14 +1,30 @@
 import { FC } from 'react';
-import { Routes, Route } from 'components/common/common';
+import { Routes, Route, Link } from 'components/common/common';
 import { AppRoute } from 'common/enums/enums';
 import { Auth } from 'components/auth/auth';
+import { Home } from 'components/auth/components/home';
+import { PrivateRoute } from '../auth/hoc/private-route';
 
 const App: FC = () => {
   return (
     <>
+      <ul className="App-navigation-list">
+        <li>
+          <Link to={AppRoute.ROOT}>Home</Link>
+        </li>
+      </ul>
+
       <Routes>
         <Route path={AppRoute.SIGN_IN} element={<Auth />} />
         <Route path={AppRoute.SIGN_UP} element={<Auth />} />
+        <Route
+          path={AppRoute.ROOT}
+          element={
+            <PrivateRoute>
+              <Home />
+            </PrivateRoute>
+          }
+        />
```

Comment:
```suggestion
        <PrivateRoute
          path={AppRoute.ROOT}
          element={<Dashboard />}
        />
```
it should be in this way